src/gfx/rgba.cpp

```diff
@@ -0,0 +1,52 @@
+
+#include "gfx/rgba.hpp"
+
+#include <assert.h>
+#include <stdint.h>
+
+#include "gfx/main.hpp" // options
+
+/*
+ * based on the Gaussian-like curve used by SameBoy since commit
+ * 65dd02cc52f531dbbd3a7e6014e99d5b24e71a4c (Oct 2017)
+ * with ties resolved by comparing the difference of the squares.
+ */
+static std::array<uint8_t, 256> reverse_curve{
```

Comment:
```suggestion
static std::array<uint8_t, 256> reverse_curve {
```

Reply:
`clang-format` disagrees.

Reply:
True, it's an initializer, not an array/struct literal being assigned.